src/Migrator.php

```diff
@@ -11,13 +11,23 @@
 
 namespace Spiral\Migrations;
 
+use Spiral\Database\Database;
 use Spiral\Database\DatabaseManager;
 use Spiral\Database\Table;
 use Spiral\Migrations\Config\MigrationConfig;
 use Spiral\Migrations\Exception\MigrationException;
 
 final class Migrator
 {
+    private const DB_DATE_FORMAT = 'Y-m-d H:i:s';
```

Comment:
@wolfy-j Could we use it for all drivers? As an possible option we should prepare and use some driver method to get this format

Reply:
This is file date format, not database one, so yes. You can use it for all drivers, upon persisting it will be converted to DateTime.